pytest_dbfixtures/utils.py

```diff
@@ -109,3 +110,29 @@ def extract_version(text):
     else:
         extracted_version = None
     return extracted_version
+
+
+def find_executable(file):
```

Comment:
Question:

Is this method even needed?

In task #107, author wrote:  `pytest-dbfixtures should trust that the user has the PATH configured correctly in their test execution environment`, to which @pwilczynskiclearcode basically agreed. 

Having executables on PATH means that we can simply call redis-server to start redis-server, and all we'd need is to check it's version for redis-server fixtures.